src/core/events/bpf/include/events.h

```diff
@@ -0,0 +1,82 @@
+#ifndef __CORE_PROBE_KERNEL_BPF_EVENTS__
+#define __CORE_PROBE_KERNEL_BPF_EVENTS__
+
+#include <vmlinux.h>
+
+/* Please keep both synced with its Rust counterpart. */
+#define EVENTS_MAX		512
+#define RAW_EVENT_DATA_SIZE	1024 - 2 /* Remove the size field */
+
+/* Please keep in sync with its Rust counterpart in crate::core::events::raw. */
+enum trace_event_owners {
+	COMMON = 1,
+	COLLECTOR_SKB_TRACKING = 2,
+};
+
+struct trace_raw_event {
+	u16 size;
+	u8 data[RAW_EVENT_DATA_SIZE];
+} __attribute__((packed));
+
+/* Please keep synced with its Rust counterpart. */
+struct trace_raw_event_section_header {
+	u8 owner;
+	u8 data_type;
```

Comment:
I don't see an  enum declared for this. The value `1` hardcoded both in the bfpf and the rust side. I assume `1` means "custom data type" but I think we should have it declared and not hardcoded.

Reply:
Oh, ok. Then I miss-interpreted it. I thought it was some kind of reference to the actual type of data that it contained, the use of `DATA_TYPE_U128` in the unit tests got me confused. It's more an "id" than a "type".